brands/amenity/cinema.json

```diff
@@ -165,6 +165,25 @@
       "name": "Cineplanet"
     }
   },
+  "amenity/cinema|Cineplex Entertainment": { 
+    "countryCodes": ["ca"],
+    "matchNames": [
+      "cineplex cinema",
+      "cineplex cinemas",
+      "cineplex entertainment",
+      "cineplex odeon",
+      "cineplex theatre",
+      "cineplex theatres",
+      "galaxy cinemas"
```

Comment:
Adding "Galaxy Cinemas" here means that iD will prompt users to upgrade those features to "Cineplex".  Is this what you want?    (If not, they need to be 2 separate entries).

Reply:
This is an old name for the company. Thought of adding as a match just in case. Does it make sense?

Reply:
I don't know whether it makes sense or not.  `matchNames` is for alternate names that are less preferred. 

iD will encourage users to upgrade them to the preferred name.  If the theater still operates some locations as the "Galaxy" brand, then this would be wrong.  But if they have changed all the locations to have "Cineplex" signs, then it would be correct.

Reply:
@bhousel  thanks for clarifying. It makes more sense to remove it then.